src/torchcodec/decoders/_video_decoder.py

```diff
@@ -88,6 +90,18 @@ def __init__(
                 f"Invalid seek mode ({seek_mode}). "
                 f"Supported values are {', '.join(allowed_seek_modes)}."
             )
+        if custom_frame_mappings:
+            if seek_mode not in ("exact", "custom_frame_mappings"):
+                raise ValueError(
+                    "While setting custom frame mappings, do not set `seek_mode`."
+                )
+            # Set seek mode to avoid exact mode scan
+            seek_mode = "custom_frame_mappings"
+        custom_frame_mappings_data = (
+            read_custom_frame_mappings(custom_frame_mappings)
+            if custom_frame_mappings is not None
+            else None
```

Comment:
Is it ever valid for `seek_mode == "custom_frame_mappings"` and `custom_frame_mappings_data is None` to both be true? If no, then we should probably raise an error here.

Actually, shouldn't line 93 mean that `custom_frame_mappings` *cannot* be `None`?